logpolicy/logpolicy.go

```diff
@@ -40,6 +41,23 @@ import (
 	"tailscale.com/version"
 )
 
+var getLogTargetOnce struct {
+	sync.Once
+	target string
+}
+
+func getLogTarget() string {
+	getLogTargetOnce.Do(func() {
+		if runtime.GOOS == "windows" {
+			getLogTargetOnce.target = getRegString("LogTarget", "")
+		} else {
```

Comment:
We try not to diverge from Linux behavior too much on Windows if we can help it.

How about we check the environment variable first, and if that's empty, then fall back to the registry on Windows?

Reply:
Done